### PR TITLE
Devex 1100 swagger doc housekeeping

### PR DESCRIPTION
**Pull request by jasonfu9966**
just cleaning up this swagger doc:
- combining "get" and "post" for /cases into one block
- adding an "items" block wherever "array" is specified as the type
- disable "try it out" button (swaggerLambda)
- formatting

known issues: swagger UI is unable to properly display nested schemas. the schema in the JSON file is correct; the schema on SwaggerUI is not.

### CODE REVIEW

File: web-api/src/swagger/swaggerLambda.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ exports.swaggerLambda = () => {
       <script>
         SwaggerUIBundle({
           dom_id: '#swagger',
-          url: '/api/swagger.json'
+          url: '/api/swagger.json',
+          supportedSubmitMethods: []
```
Comment (jasonfu9966) [inline web-api/src/swagger/swaggerLambda.js:20]: removes the "try it out!" button from the docs.

File: web-api/swagger.json [hunk 2]
```diff
@@ -412,6 +412,25 @@
...
+              "$ref": "#/definitions/case"
+            }
+          }
+        },
+        "security": [
+          {
+            "CognitoUserPool": []
+          }
+        ]
+      },
```
Comment (jasonfu9966) [inline web-api/swagger.json:433]: relocation. combined two separate "/cases" blocks.

File: web-api/swagger.json [hunk 3]
```diff
@@ -4827,31 +4802,7 @@
...
-              },
-              "sort": {
-                "type": "string"
-              },
-              "userHeading": {
-                "type": "string"
-              }
-            },
-            "required": ["showCaseNotes", "filters", "formattedCases", "formattedTrialSession", "sort", "userHeading"]
+            "$ref": "#/definitions/printableWorkingCopyData"
```
Comment (jasonfu9966) [inline web-api/swagger.json:4805]: moved this schema from here into its own definition at the bottom of the file.

**Review by jasonfu9966 — COMMENTED** [review]
see comments for explanation.

**Review by rgoshen — APPROVED** [review]

**Review by jimlerza — APPROVED** [review]

**Pull request merged**